tests/br_300_small_tables/run.sh

```diff
@@ -56,6 +76,24 @@ while [ $i -le $TABLES_COUNT ]; do
     i=$(($i+1))
 done
 
+rm -rf $RESTORE_LOG
+echo "restore 1/300 of the table start..."
+run_br restore table --db $DB  --table "sbtest100" --log-file $RESTORE_LOG -s "local://$TEST_DIR/$DB" --pd $PD_ADDR --no-schema
+restore_size=`tail -n 2 ${RESTORE_LOG} | grep "restore data size" | grep -oP '\[\K[^\]]+' | grep "restore data size" | awk -F '=' '{print $2}' | grep -oP '\d*\.\d+'`
+echo ${restore_size}
+
+diff=`echo "${backup_size}-${restore_size}*${TABLES_COUNT}" | bc`
```

Comment:
Just a hint, the current version is also fine.

In bash, You can use [Shell Arithmetic](https://www.gnu.org/software/bash/manual/html_node/Arithmetic-Expansion.html#Arithmetic-Expansion) for cleaner arithmetic expressions, like:

```suggestion
diff=$((backup_size-restore_size*TABLES_COUNT))
```